test/Chainweb/Test/RestAPI/Utils.hs

```diff
@@ -119,28 +120,42 @@ repeatUntil test action = retrying testRetryPolicy
     (\_ b -> not <$> test b)
     (const action)
 
+
 -- | Calls to /local via the pact local api client with retry
 --
-local
+localWithQueryParams
     :: ChainId
     -> ClientEnv
+    -> Bool
+    -> Maybe Word64
     -> Command Text
     -> IO (CommandResult Hash)
-local sid cenv cmd =
+localWithQueryParams sid cenv pf rd cmd =
     recovering testRetryPolicy [h] $ \s -> do
       debug
         $ "requesting local cmd for " <> take 19 (show cmd)
         <> " [" <> show (view rsIterNumberL s) <> "]"
 
-      -- send a single spv request and return the result
+      -- send a single local request and return the result
       --
-      runClientM (pactLocalApiClient v sid cmd) cenv >>= \case
+      runClientM (pactLocalWithQueryApiClient v sid pf rd cmd) cenv >>= \case
         Left e -> throwM $ LocalFailure (show e)
         Right t -> return t
   where
     h _ = Handler $ \case
-      LocalFailure _ -> return True
-      _ -> return False
+      LocalFailure _ -> pure False
```

Comment:
Why the boolean flip? Was this a bug?

Reply:
This was a stale change - nice catch. Turns out we never trip it because we don't hit any retry cases in the local tests anymore.